scripts/obtain/dotnet-install.ps1

```diff
@@ -366,6 +367,29 @@ function DownloadFile([Uri]$Uri, [string]$OutPath) {
     }
 }
 
+function Get-Runtime-Version-From-Dotnet-Package([System.IO.Compression.ZipArchive]$Zip) {
```

Comment:
I think we can give repo's an option to specify the version as a default parameter and if it isn't specified then you can find the highest version in the installed cli.